css-ui-4/Overview.bs

```diff
@@ -707,7 +707,7 @@ Values have the following meanings:
 			If the values are unspecified,
 			then the intrinsic hotspot defined inside the image resource itself is used.
 			If both the values are unspecific and the referenced cursor has no defined hotspot,
-			the effect is as if a value of "0 0" were specified.
+			the effect is as if a value of "0 0" was specified.
```

Comment:
Not a native speaker, but this seems an unreal conditional, so "were" was correct?
https://www.grammarly.com/blog/conditional-sentences-was-instead-of-were/

Reply:
I hadn't heard of 'unreal conditional' before, but I wouldn't read "0 0" as being implausible here.

As a native speaker, I would always use 'was' here due to the non-plural "a value".

The following might also have worked: "the effect is as if the values `0 0` were specified."

Reply:
I am also not a native speaker, though I looked up the grammar about unreal conditionals before I created the PR and came to the same conclusion as Eric.

The page you referred to clearly states "When you’re talking about a possibility that did happen or might be true, use “was” and “were” as you normally would."

So, according to that, "was" is correct.

Sebastian

Reply:
Well, we are under the assumption that "both the values are unspecific", so it's impossible for "0 0" to be specified. Then it seems an unreal conditional to me, but it's not like this matters much.